In docker compose mode, use nginx load balancing (#10434)
Minio-io/minio: docs/orchestration/docker-compose/README.md

docker-compose.exe up
```

> Distributed instances is now accessible on the host at ports 9000, proceed to access the Web browser at http://127.0.0.1:9000/. Here 4 minio server instances are reverse proxyed through Nginx load balancing.

Reviewer: ```suggestion
Distributed instances are now accessible on the host at ports 9000, proceed to access the Web browser at http://127.0.0.1:9000/. Here 4 MinIO server instances are reverse proxied through Nginx load balancing.
```

Author: Resolved,thank you for your review. @harshavardhana

* There are 4 minio distributed instances created by default. You can add more MinIO services (up to total 16) to your MinIO Compose deployment. To add a service
  * Replicate a service definition and change the name of the new service appropriately.
  * Update the command section in each service.

>   * Add new minio server instance to upstream directive of the nginx.conf file.

Reviewer: ```suggestion
  * Add a new MinIO server instance to the upstream directive in the Nginx configuration file.
```

  Read more about distributed MinIO [here](https://docs.min.io/docs/distributed-minio-quickstart-guide).

> * Normally, Docker Compose runs on a single host, so there is no need to export ports.

Reviewer: ```suggestion
```

Remove this
